driver/src/price_finding/naive_solver.rs

```diff
@@ -497,12 +472,110 @@ pub mod tests {
                 buy_amount: 19960,
             },
         ];
+        let state = create_account_state_with_balance_for(&orders);
 
-        let mut solver = NaiveSolver::new(Some(Fee {
+        let fee = Some(Fee {
             token: 2,
             percentage: 0.001,
-        }));
+        });
+        let mut solver = NaiveSolver::new(fee.clone());
         let res = solver.find_prices(&orders, &state).unwrap();
         assert_eq!(res.executed_sell_amounts, [0, 0]);
```

Comment:
Here, for example, we only check executed sell amounts. But we should also check buy amounts and prices.